grails-app/views/phylo/show.gsp

```diff
@@ -315,28 +330,20 @@
         },
         downloadSummaryUrl: '${createLink(controller: "phylo", action:"getHabitat" )}/?download=true',
         biocacheOccurrenceDownload: 'http://biocache.ala.org.au/ws/occurrences/index/download',
-        downloadReasonsUrl: config.downloadReasonsUrl
+        downloadReasonsUrl: config.downloadReasonsUrl,
+        records: records
     });
 
-    var records = new Records({
-        id: 'recordsForm',
-        template: $('#templateOccurrence').html(),
-        uploadUrl: '${createLink(controller: 'ala', action: 'uploadData')}?phyloId=${phyloInstance.id}',
-        indexingStatusUrl: "${createLink(controller: 'sandbox', action: 'checkStatus')}",
-        sampleFile: "${createLink(controller: 'artifacts', action: 'occurrenceRecords.csv')}",
-        dataresrouceInfoUrl: "${createLink(controller: 'sandbox', action: 'dataresourceInfo')}?phyloId=${phyloInstance.id}",
-        dataresourceListUrl: '${createLink(controller: 'ala', action: 'getRecordsList')}?phyloId=${phyloInstance.id}',
-        map : map,
-        pj: pj,
-        selectResourceOnInit: true,
-        initResourceId: -1
-    });
 
     $( "#tabs" ).tab('show');
     // a fix to display map tiles properly. Without it map is grey colored for majority of area.
     $("body").on("shown.bs.tab", "#mapTab", function() {
         map.invalidateSize();
     });
+    $("body").on("shown.bs.tab", "#habitatTab", function() {
```

Comment:
This fixes the habitat chart formatting issue that occurs when the charts are drawn and another tab is active. Probably a better way to do this.